@here/harp-mapview/lib/MapView.ts

```diff
@@ -3439,6 +3492,10 @@ export class MapView extends THREE.EventDispatcher {
 
         this.checkCopyrightUpdates();
 
+        setTimeout(() => {
```

Comment:
Not sure if this can really happen, but I think we need to check whether the timeout from the last frame was even processed.